src/adaptations/device-layer/include/Weave/DeviceLayer/SoftwareUpdateManager.h

```diff
@@ -52,109 +52,185 @@ class SoftwareUpdateManager
     {
         kState_Idle                 = 1,
         kState_ScheduledHoldoff     = 2,
-        kState_Prepare              = 3,
+        kState_PrepareQuery         = 3,
         kState_Query                = 4,
-        kState_Download             = 5,
-        kState_Install              = 6,
+        kState_PrepareImageStorage  = 5,
+        kState_Download             = 6,
+        kState_Install              = 7,
 
-        kState_ApplicationManaged   = 7,
+        kState_ApplicationManaged   = 8,
 
         kState_MaxState
     };
 
+    /**
+     * API events generated by the \c SoftwareUpdateManager object.
```

Comment:
\c  ?

Reply:
doxygen syntax to display following word in "typewriter" font.